src/components/AuthenticatedAppHeader/NoticiationPaneDisplayText.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import Text from '../Text';
+import Link from '../Link';
+
+export default function NotificationPaneDisplayText({
+  currentNotificationSchema,
+  userName,
+  user1Name,
+  user2Name,
+  yourIndividualName,
+  theirIndName,
+  theirIndividualGuid,
+  formattedDeadline,
+}) {
+  const intl = useIntl();
+  return (
+    <Text style={{ maxWidth: 200, margin: '0 20px' }}>
```

Comment:
Instead of using intl, you should be able to just do 
```
<Text id={currentNotificationSchema?.notificationMessage} values={{ userName, ... }} />
``

Reply:
Cool! That works!